diff --git a/build/make/configure.sh b/build/make/configure.sh
index a65d395..579e0d6 100755
--- a/build/make/configure.sh
+++ b/build/make/configure.sh
@@ -172,6 +172,7 @@ add_cflags() {
 
 
 add_cflags_only() {
+    log_echo "Adding $@ to CFLAGS"
     CFLAGS="${CFLAGS} $@"
 }
 
diff --git a/build/make/rtcd.pl b/build/make/rtcd.pl
index 18ee80d..a3dc890 100755
--- a/build/make/rtcd.pl
+++ b/build/make/rtcd.pl
@@ -385,6 +385,6 @@ if ($opts{arch} eq 'x86') {
 } elsif ($opts{arch} eq 'armv6') {
   @ALL_ARCHS = filter(qw/edsp media/);
   arm;
-} elsif ($opts{arch} eq 'armv7') {
+} elsif ($opts{arch} eq 'armv7' || $opts{arch} eq 'armv7s') {
   @ALL_ARCHS = filter(qw/edsp media neon_asm neon/);
   @REQUIRES = filter(keys %required ? keys %required : qw/media/);
